packages/core-ui/lib/core/Grid/index.tsx

```diff
@@ -0,0 +1,38 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: Apache-2.0 */
+
+import { GridProps } from "../types";
+import "./index.scss";
+
+export const Grid = ({
+  alignItems,
+  children,
+  column,
+  expanded,
+  justify,
+  lg,
+  md,
+  row,
+  sm,
+  style,
+}: GridProps) => {
+  const isRow: boolean = row || !column;
+
+  const classes: string =
+    // Clarify if its a column or a row
+    (!isRow ? "column" : "row") +
```

Comment:
There is a utility called `valEmpty` that will check if a variable is truthy, and have an empty string if it is not. So:

```
(!isRow && sm ? ` ${"sm-" + sm}` : "") +
```

Can become:
```
valEmpty(!isRow && !!sm, `sm-${sm}`)
```
And so forth. Not a huge impact but just keeps it looking like the code is written by one dev team :)